spec/config/options_spec.rb

```diff
@@ -0,0 +1,79 @@
+RSpec.describe "options.yml" do
+  let(:options) { ManageIQ::RPMBuild::OPTIONS }
+
+  nightly_versions = [
+    "13.1.0-20211208000053",
```

Comment:
We are going to have to probably change the branching script to deal with these specs, since they are branch-specific, otherwise these are going to break next time we branch.

Reply:
I've got an idea, I'll make a master PR

Reply:
https://github.com/ManageIQ/manageiq-rpm_build/pull/231